25.md

```diff
@@ -7,20 +7,16 @@ Reactions
 
 `draft` `optional`
 
-A reaction is a `kind 7` event that is used to react to other events.
+A reaction is a `kind 7` event that is used to indicate user reactions to other events. A
+reaction's `content` field MUST include a single character indicating the value of the reaction.
+Reactions MUST NOT be used to support application-level semantics, they are for user-generated
+content only.
 
-The generic reaction, represented by the `content` set to a `+` string, SHOULD
-be interpreted as a "like" or "upvote".
+A reaction with `content` set to `+` or an empty string MUST be interpreted as a "like" or "upvote".
+A reaction with `content` set to `-` MUST be interpreted as a "dislike" or "downvote".
 
-A reaction with `content` set to `-` SHOULD be interpreted as a "dislike" or
-"downvote". It SHOULD NOT be counted as a "like", and MAY be displayed as a
-downvote or dislike on a post. A client MAY also choose to tally likes against
-dislikes in a reddit-like system of upvotes and downvotes, or display them as
-separate tallies.
-
-The `content` MAY be an emoji, or [NIP-30](30.md) custom emoji in this case it MAY be interpreted as a "like" or "dislike",
-or the client MAY display this emoji reaction on the post. If the `content` is an empty string then the client should
-consider it a "+".
+A reaction with `content` set to an emoji or [NIP-30](30.md) custom emoji SHOULD NOT be interpreted
+as a "like" or "dislike". Clients MAY instead display this emoji reaction on the post.
```

Comment:
Clients SHOULD (not MAY) display the emoji if present.

Reply:
Just keeping the original content of the NIP. I think MAY is actually correct, because you may filter by emoji and display related posts, in which case display would be redundant.